### ISSUE TITLE
Create Display Pages

### ISSUE THREAD

**Issue by Vipul999ujawane** [Vipul999ujawane/acad-search#2]

**Comment by kurianbenoy** [comment]
Please do assign me this issue

**Comment by kurianbenoy** [comment]
I agree that front end is kind of bad now, what kind of Display pages do you need?
We definitely need a new home page and in a search bar , we can sort document

**Comment by Vipul999ujawane** [comment]
Yeah, that's what I am looking for. The display pages are for showing the documents.

**Comment by pavangudiwada** [comment]
can i know what kind of display pages?

**Comment by Vipul999ujawane** [comment]
A page where after going through the year and dept thing, we'll be able to access the documents. I meant make the file "file.html" better

**Comment by pavangudiwada** [comment]
Im kind of new to this but ill do everything i can to make it better

**Comment by Vipul999ujawane** [comment]
Its better if you talk through the gitter channel.

**Comment by pavangudiwada** [comment]
ok
